enzyme/Enzyme/MLIR/Interfaces/GradientUtils.cpp

```diff
@@ -38,7 +41,7 @@ mlir::enzyme::MGradientUtils::MGradientUtils(
       originalToNewFn(originalToNewFn_),
       originalToNewFnOps(originalToNewFnOps_),
       invertedPointers(invertedPointers_) {
-
+  
```

Comment:
Let's avoid spurious whitespace changes as much as possible. They make it harder to track code evolution.